README.md

```diff
@@ -5,12 +5,181 @@
 For more information about ROS 2 interfaces, see [docs.ros.org](https://docs.ros.org/en/rolling/Concepts/About-ROS-Interfaces.html).
 
 ## Messages (.msg)
+Collection of telemetry (feedback) from buoy sensors and microcontrollers.
 
-TODO
+### Telemetry
+Telemetry data from sensors available per microcontroller
+
+- type: `BCRecord`  
+  topic: `/bc_record`  
```

Comment:
I think it's not very common to document the topic together with the messages, because in theory the messages could be used by different topics. But since we're initially targeting a specific use case, I think this can be helpful.

Reply:
I agree completely. I was debating whether to put topics in. Like you said, we are initially targeting our specific wave energy buoy. So, I was thinking of this as more of an ICD for our system than a generic package.